com/edocti/jintro/lab02/Palindrome.java

```diff
@@ -0,0 +1,36 @@
+package com.edocti.jintro.lab02;
+import java.util.Scanner;
+public class Palindrome {
+
+    //Method to check
+    private static boolean isPalindrome(String s) {
+        // if length is 0 or 1 then String is palindrome
+        int N = s.length();
+        if (N == 0 || N == 1) {
+            return true;
+        }			
+        if(s.charAt(0) == s.charAt(N-1)) {
+        /* check for first and last char of String:
+        * if they are same then do the same thing for a substring
+        * with first and last char removed. and carry on this
+        * until you string completes or condition fails
+        * Function calling itself: Recursion
+        */
+        return isPalindrome(s.substring(1, N-1));
```

Comment:
This is OK as an exercise for accustoming yourself with recursive calls. Good that you tried it and understood it.  :+1: However, please note that the iterative version has better performance and it's easier. Optionally, you can add a new method for the iterative version.